tokenserver/tests/test_sql.ini

```diff
@@ -15,9 +15,8 @@ secrets.master_secrets = "abcdef"
                          "123456"
 node_type_patterns =
   example:*example.com
-spanner_entry  = https://spanner.example.com
 spanner_node_id = 800
-migrate_new_user_percentage=0
+migrate_new_user_percentage=1
```

Comment:
By setting this in the config file rather than in code, we test that the loading-from-config-file logic also works as expected.

Reply:
hrm, this will fire on every sql test, though. Going to move this to the actual test to limit future swearing.

Reply:
Does it matter though? None of the behaviour of the other tests should be affected by whether the user goes to the (fake) spanner node or not.

Alternately, maybe we should move this into the backend tests under `tests/assignment/` if it's specific to a particular backend (which may be where you had it originally).

Reply:
Hi Ryan! Have you met my good friend Murphy? He likes to drop by late at night in the future while trying to figure out why some tests are going weird. ;)

Murphy and I go way, way back and I've learned to look for unlocked doors, loose window jams, and unsecured floor boards he might use to get in.